### DISCUSSION TITLE
What is `LogMicro` in default-config.json ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#1232] [General]
Hi,

I don't find the description in https://github.com/evilsocket/opensnitch/wiki/Configurations

if confirmed missing, I'll turn this topic into a an Issues. (I can take care of updating the wiki, just need the info :) )

**Comment by gustavo-iniguez-goya** [comment]
LogMicro and LogUTC print the logs with microseconds and in UTC format respectively. By default the date format is printed as the system local time.

**Reply by BobSquarePants** [reply]
PR created ;) https://github.com/evilsocket/opensnitch/pull/1241

**Reply by gustavo-iniguez-goya** [reply]
thank you!
